tools/ci/src/args.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Copy, Clone, PartialEq, Eq)]
+pub struct Args {
+    pub keep_going: bool,
+    pub test_threads: Option<u8>,
+}
```

Comment:
This is missing the `build_jobs` flag.

Reply:
It wouldn't be used if it were added here at the moment, but I should propagate that downward, too.